tests/plugins/notification/notification.js

```diff
@@ -440,5 +440,16 @@ bender.test( {
 		assert.isTrue( listener.calledOnce );
 
 		assertNotifications( editor, [ { message: 'Foo', type: 'info', alert: true } ] );
+	},
+
+	// #1057
+	'test Notification should not leak the global scope': function() {
+		// Web Notification Api is not supported by IE.
+		if ( typeof Notification === 'undefined' ) {
+			assert.ignore();
```

Comment:
Actually if `window.Notification` is undefined, the test should pass – as it clearly shows that `Notification` hasn't leaked (no variable = no leakage).